serving/docker/partition/partition.py

```diff
@@ -276,6 +283,36 @@ def autoawq_quantize(self):
         awq_model.save_quantized(output_path)
         tokenizer.save_pretrained(output_path)
 
+    def autofp8_quantize(self):
+        sys.path.append(PYTHON_CACHE_DIR)
+        from djl_python.properties_manager.hf_properties import HuggingFaceProperties
+
+        output_path = self.properties['option.save_mp_checkpoint_path']
+        properties = remove_option_from_properties(self.properties)
+        hf_configs = HuggingFaceProperties(**properties)
+        tokenizer = init_hf_tokenizer(hf_configs.model_id_or_path, hf_configs)
```

Comment:
Seems like 287-293 are duplicated from AWQ path. Please refactor to reduce duplication.